### PR TITLE
ARTEMIS-3396 Convert bytes properties to String for OpenWire

### PR DESCRIPTION
**Pull request by brusdev**

### CODE REVIEW

**Review by franz1981 — APPROVED** [review]
LGTM!

File: artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverter.java [hunk 1]
```diff
@@ -963,13 +965,14 @@ private static void setAMQMsgObjectProperties(final ActiveMQMessage amqMsg,
          try {
             if (prop instanceof SimpleString) {
                amqMsg.setObjectProperty(keyStr, prop.toString());
+            } else if (prop instanceof byte[]) {
+               ActiveMQServerLogger.LOGGER.propertyConvertedToString(keyStr, "byte[]");
```
Comment (clebertsuconic) [inline artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverter.java:969]: I'm not sure you should really log a warn for this. This looks regular business to me. it should just to it.

File: artemis-protocols/artemis-openwire-protocol/src/test/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverterTest.java [hunk 2]
```diff
@@ -91,4 +92,20 @@ public void createMessageDispatch() throws Exception {
...
+      ICoreMessage coreMessage = new CoreMessage().initBuffer(8);
+      coreMessage.putBytesProperty(bytesPropertyKey, "TEST".getBytes());
+
+      MessageReference messageReference = new MessageReferenceImpl(coreMessage, Mockito.mock(Queue.class));
+      AMQConsumer amqConsumer = Mockito.mock(AMQConsumer.class);
+      Mockito.when(amqConsumer.getOpenwireDestination()).thenReturn(destination);
+
+      MessageDispatch messageDispatch = OpenWireMessageConverter.createMessageDispatch(messageReference, coreMessage, openWireFormat, amqConsumer, nodeUUID);
+
+      Assert.assertTrue(messageDispatch.getMessage().getProperty(bytesPropertyKey) instanceof String);
```
Comment (clebertsuconic) [inline artemis-protocols/artemis-openwire-protocol/src/test/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverterTest.java:109]: Can you add a test where it's actually sending a message and converting to OpenWire and AMQP?

Comment (brusdev) [reply artemis-protocols/artemis-openwire-protocol/src/test/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverterTest.java]: I have just added the testBinaryPropertyConversionToString test converting from AMQP to OpenWire.

**Pull request merged**